src/core/MOM.F90

```diff
@@ -2339,8 +2341,13 @@ subroutine register_diags(Time, G, GV, CS, ADp)
 
   thickness_units = get_thickness_units(GV)
   flux_units      = get_flux_units(GV)
-  T_flux_units    = get_tr_flux_units(GV, "degC")
-  S_flux_units    = get_tr_flux_units(GV, "psu")
+  T_flux_units    = trim("Watts/m2")
```

Comment:
CMIP units are "W m-2". I'll change it when merging.

http://clipc-services.ceda.ac.uk/dreq/index/units.html